docs/dev/documentation.md

```diff
@@ -32,7 +32,7 @@ def example(param1: int, param2: str) -> bool:
     """
 ```
 
-* The order of sections is (1) Args (2) Returns (3) Raises (4) Examples
+* The order of sections is 1. Args 2. Returns 3. Raises 4. Examples
```

Comment:
Which benefit does this have? I consider the old variant more readable.

Reply:
How about a compromise?

> * The order of sections is 1) Args 2) Returns 3) Raises 4) Examples

Reply:
Does this really improve readability? I consider the original variant the most readable.

Reply:
Reverted.